packages/near-membrane-dom/src/window.ts

```diff
@@ -238,3 +238,22 @@ export function tameDOM(
     env.remap(blueRefs.WindowPropertiesProto, WindowPropertiesDescriptors);
     env.remap(blueRefs.WindowProto, blueRefs.WindowProtoDescriptors);
 }
+
+export function linkUnforgeables(
+    env: VirtualEnvironment,
+    blueGlobalThis: Window & typeof globalThis
+) {
+    // The test of instance of event target is important to discard environments in which
+    // a fake window (e.g. jest) is not following the specs, and can break this
+    // membrane.
+    if (blueGlobalThis.EventTarget && blueGlobalThis instanceof EventTarget) {
+        // window.document
+        env.link(`document`);
+        // window.__proto__ (aka Window.prototype)
```

Comment:
this is where things go south for me, the `__proto__` might be problematic in node, but this code is not for node, it is for a window, in which case `__proto__` is safe for now.

Reply:
This is super 😎